server/pulp/server/webservices/views/consumers.py

```diff
@@ -198,6 +276,42 @@ def put(self, request, consumer_id):
         return generate_json_response_with_pulp_encoder(consumer)
 
 
+class ConsumerSearchView(search.SearchView):
+    """
+    This view provides GET and POST searching for Consumers.
+    """
+    optional_fields = ['details', 'bindings']
+    response_builder = staticmethod(generate_json_response_with_pulp_encoder)
```

Comment:
Do they pretty much all use "_with_pulp_encoder"? If so, maybe we can just drop this class attr and just call that one all the time…